app/assets/javascripts/course/assessment/submission/submission.js

```diff
@@ -164,8 +164,16 @@
       $('#' + identifier).find('textarea.code').ace();
     });
 
-    // Show the first tab on page load.
-    $(MULTI_QUESTION_ASSESSMENT_SELECTOR + '.tab-header a:first').tab('show');
+    // Check the URL for a step parameter
+    var url = document.location.toString();
+    if (url.match('step=')) {
```

Comment:
Possible to abstract this out into functions? 

Something like `renderTab(selector, index)` and `getParamFromURL(url, param)`. Then we get (psuedo code):

```
idx = getParamFromURL(url,param) || 1
renderTab(selector, idx)
```